experimental/web_dashboard/README.md

```diff
@@ -1,6 +1,17 @@
 # web_dashboard
 
-(In progress) A dashboard app for the web, built with Flutter.
+**In progress** 
+
+A dashboard app that displays daily entries.
+
+1. How to use an AdaptiveScaffold adaptive layout for large, medium, and small
```

Comment:
Is this the goals section, similar to the ones we have for other apps?

Reply:
yeah, we should probably make this a proper section in the future.